Formula/erlang@22.rb

```diff
@@ -26,13 +27,13 @@ class ErlangAT22 < Formula
   depends_on "wxwidgets" # for GUI apps like observer
 
   resource "man" do
-    url "https://www.erlang.org/download/otp_doc_man_22.3.tar.gz"
-    sha256 "43b6d62d9595e1dc51946d55c9528c706c5ae753876b9bf29303b7d11a7ccb16"
+    url "https://github.com/erlang/otp/releases/download/OTP-22.3.4.23/otp_doc_man_22.3.4.23.tar.gz"
```

Comment:
It's a bit hard to decipher, but is the wrong man the cause of the error?

Reply:
I have absolutely no idea what is wrong now, it fails even with 22.3.4.25.

Reply:
And yes, it does fail on something with the man, but AFAICT it is supposed to just extract the tarball and the directory structure of the traball is the same as the one it is not able to download from erlang.org.

Reply:
Error may mean that directory structure of tarballs is different:
```
cp -pR /private/tmp/d20220425-68547-1htdmh6/man/. /private/tmp/erlangA22--man-20220425-68547-1jaap4g/man
chmod -Rf +w /private/tmp/d20220425-68547-1htdmh6
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - man
```

The corresponding install:
```rb
(lib/"erlang").install resource("man").files("man")
```

Reply:
I already did and run it locally, fails.

Reply:
But I do not understand why, because it does contain the `man` directory tree and nothing else. The archive used until now has exactyl the same structure.

Reply:
The original tarball has `COPYRIGHT`, etc files which means it unpacks as is.

The GitHub tarball only has a single `man` directory, which I think Homebrew compacts and removes the intermediary directory.

Reply:
so the solution is to make it move the individual `manX` subdirs?

Reply:
OK, it seems to work and I can't find any suggestion on how to avoid the directory tree flattening in the documentation, so updating the PR this way.